OnlineStore.BLL/Exceptions/UserNotFoundException.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OnlineStore.BLL.Exceptions
+{
+	public class UserNotFoundException : Exception
+	{
+		public UserNotFoundException() { }
```

Comment:
its better to pass as constructor parameters string with exception, because those exception messages that you wrote in middleware may not always describe error in details. check it in other exception classes